docs/Computation Pipeline.md

```diff
@@ -175,6 +175,7 @@ This effectively boils down to a choice of what weighting to use. The simplest w
 | 2023-07-06T05:00 | 5        | 14.3   | west-us  | 500  | 1.57        |
 | 2023-07-06T10:00 | 5        | 14.3   | west-us  | 500  | 1.57        |
 | 2023-07-06T15:00 | 5        | 20     | west-us  | 490  | 2.9         |
+| 2023-07-06T20:00 | 5        | 0      | west-us  | 0    | 0           |
```

Comment:
Hi @Urhengulas, first of all, thank you for your contribution to our project!

@jawache what should be values for `Energy, Grid, and Operational` ?
BTW I guess yaml sample below should be updated accordingly

Reply:
I think it makes sense if this is all 0, because in the previous note you say the following:

> The output impact metric is effectively zero if no impact metric overlaps a given time and duration. It means nothing was running then, so there could not be any emissions from the component.